boto/utils.py

```diff
@@ -860,11 +864,15 @@ def notify(subject, body=None, html_body=None, to_string=None,
 
 
 def get_utf8_value(value):
-    if not six.PY2 and isinstance(value, bytes):
-        return value
+    # if not six.PY2 and isinstance(value, bytes):
+    #     return value
```

Comment:
Commented code should probably be removed.

Reply:
Got it, removed